### ISSUE TITLE
NT2/Boost.SIMD status

### ISSUE THREAD

**Issue by jfalcou** [RcppCore/RcppNT2#1]
This is more a note than a real issue.

FYI, I'm the main author of NT2 and Boost.SIMD.
The SIMD features from NT2 you use in RcppNT2 has been migrated to the autonomous Boost.SIMD repository over https://github.com/NumScale/boost.simd. The API is stabilizing and the doc is far better. As the old NT2 code base may become unsupported for a while, if this project is still of some interest, I would advise for migrating to the new Boost.SIMD.

Feel free to contact me for more info :)

**Comment by kevinushey** [comment]
Thanks! We'll take a look at moving to Boost.SIMD next time we're working on RcppNT2.

(thanks for your great work on Boost.SIMD + NT2!)

**Comment by bobjansen** [comment]
The link on the [project page](https://rcppcore.github.io/RcppNT2/) has stopped working as @jfalcou's repository doesn't exist anymore (forks are available).

**Comment by eddelbuettel** [comment]
Yes, thanks, can do/should do.  Can you please add one or two of those forks here and I'll edit the page accordingly?

**Comment by bobjansen** [comment]
Going purely by commit count, DDG and Google give this one as the best option: 

https://github.com/jtlap/nt2 

or take this one:

https://github.com/bobjansen/nt2

I promise I won't delete it without a heads up :)

**Comment by eddelbuettel** [comment]
Thanks, sounds good.  I had (my mistake) thought we were on the Rcpp Gallery repo for this.  As this is more @kevinushey 's repo I'll let him make that call.

**Comment by jfalcou** [comment]
The jtlap one is probably the best.
As for the project itself, it''l be restarted later this year.
